linux_files/setup

```diff
@@ -412,88 +413,88 @@ if (whiptail --title "fcitx" --yesno "Would you like to install fcitx for improv
-fi
-
-if [[ $FCCHOICE == *"rime"* ]] ; then
-    sudo apt install fcitx-rime -y
-fi
+    if [[ $FCCHOICE == *"sunpinyin"* ]] ; then
+        sudo apt install fcitx-sunpinyin -y
+    fi
 
-if [[ $FCCHOICE == *"googlepinyin"* ]] ; then
-    sudo apt install fcitx-googlepinyin -y
-fi
+    if [[ $FCCHOICE == *"libpinyin"* ]] ; then
+        sudo apt install fcitx-libpinyin -y
+    fi
 
-if [[ $FCCHOICE == *"chewing"* ]] ; then
-    sudo apt install fcitx-chewing -y
-fi
+    if [[ $FCCHOICE == *"rime"* ]] ; then
+        sudo apt install fcitx-rime -y
+    fi
 
-if [[ $FCCHOICE == *"mozc"* ]] ; then
-    sudo apt install fcitx-mozc -y
-fi
+    if [[ $FCCHOICE == *"googlepinyin"* ]] ; then
+        sudo apt install fcitx-googlepinyin -y
+    fi
 
-if [[ $FCCHOICE == *"kkc"* ]] ; then
-    sudo apt install fcitx-kkc fcitx-kkc-dev -y
-fi
+    if [[ $FCCHOICE == *"chewing"* ]] ; then
+        sudo apt install fcitx-chewing -y
+    fi
 
-if [[ $FCCHOICE == *"hangul"* ]] ; then
-    sudo apt install fcitx-hangul -y
-fi
+    if [[ $FCCHOICE == *"mozc"* ]] ; then
+        sudo apt install fcitx-mozc -y
+    fi
 
-if [[ $FCCHOICE == *"unikey"* ]] ; then
-    sudo apt install fcitx-unikey -y
-fi
+    if [[ $FCCHOICE == *"kkc"* ]] ; then
+        sudo apt install fcitx-kkc fcitx-kkc-dev -y
+    fi
 
-if [[ $FCCHOICE == *"sayura"* ]] ; then
-    sudo apt install fcitx-sayura -y
-fi
+    if [[ $FCCHOICE == *"hangul"* ]] ; then
+        sudo apt install fcitx-hangul -y
+    fi
 
-if [[ $FCCHOICE == *"tables"* ]] ; then
-    sudo apt install fcitx-table fcitx-table-all -y
-fi
+    if [[ $FCCHOICE == *"unikey"* ]] ; then
+        sudo apt install fcitx-unikey -y
+    fi
 
-  if (whiptail --title "fcitx-autostart" --yesno "Would you like fcitx-autostart to run each time you open WLinux? WARNING: Requires an X server to be running or it will generate errors." 8 70) then
-    echo "Placing fcitx-autostart in /etc/profile"
-    sudo sh -c 'echo "fcitx-autostart" >> /etc/profile'
-   else
-    echo "Skipping fcitx-autostart"
-    whiptail --title "Note about fcitx-autostart" --msgbox "You will need to run $ fcitx-autostart to enable fcitx before running GUI apps." 8 85
-   fi
+    if [[ $FCCHOICE == *"sayura"* ]] ; then
+        sudo apt install fcitx-sayura -y
+    fi
 
-   echo "Configuring dbus machine id"
-   sudo sh -c "dbus-uuidgen > /var/lib/dbus/machine-id"
+    if [[ $FCCHOICE == *"tables"* ]] ; then
+        sudo apt install fcitx-table fcitx-table-all -y
+    fi
 
-   echo "Setting environmental variables"
-   export GTK_IM_MODULE=fcitx
-   export QT_IM_MODULE=fcitx
-   export XMODIFIERS=@im=fcitx
-   export DefaultIMModule=fcitx
+    echo "Setting environmental variables"
+    export GTK_IM_MODULE=fcitx
+    export QT_IM_MODULE=fcitx
+    export XMODIFIERS=@im=fcitx
+    export DefaultIMModule=fcitx
+
+    echo "Saving environmental variables to /etc/profile"
+    sudo sh -c 'echo "export QT_IM_MODULE=fcitx" >> /etc/profile'
```

Comment:
Now that you are touching this part, I would recommend write all the modifications to /etc/profile.d/something instead of modifying /etc/profile itself. So you can run the script over and over again without stacking up the exports